src/index.html

```diff
@@ -17,6 +20,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="center"></div>
+      <div class="hand">
+        <div class="hand__minute"></div>
+        <div class="hand__second"></div>
+      </div>
```

Comment:
Consider adding `aria-live="assertive"` attribute to the element that will display the stopwatch time. This ensures that changes to the stopwatch's time are communicated to screen readers in a timely manner.